src/modules/75-cd/components/PipelineSteps/CloudFormation/CloudFormationHelper.tsx

```diff
@@ -0,0 +1,205 @@
+  }
+  if (type === Connectors.BITBUCKET) {
+    return buildBitbucketPayload
+  }
+  if (type === Connectors.GITLAB) {
+    return buildGitlabPayload
+  }
+  if (type === Connectors.AWS) {
+    return buildAWSPayload
+  }
+  return () => ({})
+}
+
+export const GetNewConnector = (
+  connectorType: string,
+  isEditMode: boolean,
+  setIsEditMode: (bool: boolean) => void,
+  accountId: string,
+  projectIdentifier: string,
+  orgIdentifier: string,
+  name: string
+): JSX.Element | null => {
+  console.log('connectorType', connectorType)
+  switch (connectorType) {
+    case Connectors.GIT:
+      return (
+        <StepGitAuthentication
+          name={name}
+          onConnectorCreated={() => {
+            // Handle on success
+          }}
+          isEditMode={isEditMode}
+          setIsEditMode={setIsEditMode}
+          connectorInfo={undefined}
+          accountId={accountId}
+          orgIdentifier={orgIdentifier}
+          projectIdentifier={projectIdentifier}
+        />
+      )
+    case Connectors.GITHUB:
+      return (
+        <StepGithubAuthentication
+          name={name}
+          onConnectorCreated={() => {
+            // Handle on success
+          }}
+          isEditMode={isEditMode}
+          setIsEditMode={setIsEditMode}
+          connectorInfo={undefined}
+          accountId={accountId}
+          orgIdentifier={orgIdentifier}
+          projectIdentifier={projectIdentifier}
+        />
+      )
+    case Connectors.GITLAB:
+      return (
+        <StepGitlabAuthentication
+          name={name}
+          identifier={CONNECTOR_CREDENTIALS_STEP_IDENTIFIER}
+          onConnectorCreated={() => {
+            // Handle on success
+          }}
+          isEditMode={isEditMode}
+          setIsEditMode={setIsEditMode}
+          connectorInfo={undefined}
+          accountId={accountId}
+          orgIdentifier={orgIdentifier}
+          projectIdentifier={projectIdentifier}
+        />
+      )
+    case Connectors.BITBUCKET:
+      return (
+        <StepBitbucketAuthentication
+          name={name}
+          onConnectorCreated={() => {
+            // Handle on success
+          }}
+          isEditMode={isEditMode}
+          setIsEditMode={setIsEditMode}
+          connectorInfo={undefined}
+          accountId={accountId}
+          orgIdentifier={orgIdentifier}
+          projectIdentifier={projectIdentifier}
+        />
+      )
+    case Connectors.AWS:
+      return (
+        <StepAWSAuthentication
+          name={name}
+          accountId={accountId}
+          orgIdentifier={orgIdentifier}
+          projectIdentifier={projectIdentifier}
+          connectorInfo={undefined}
+          isEditMode={isEditMode}
+          setIsEditMode={setIsEditMode}
+        />
+      )
+  }
+  return null
+}
+
+const formatPaths = (paths: any) => map(paths, (item: string) => ({ path: item }))
+
+export const FormatFilePaths = (values: any, isParam: boolean, index: number) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232213)
```suggestion
export const FormatFilePaths = (values: unknown, isParam: boolean, index: number) => {
```